skipper.go

```diff
@@ -1203,9 +1200,19 @@ func listenAndServeQuit(
 	if err != nil {
 		return err
 	}
+	serveTLS := tlsConfig != nil
+
+	address := o.Address
+	if address == "" {
+		if serveTLS {
+			address = ":https"
+		} else {
+			address = ":http"
+		}
+	}
 
 	srv := &http.Server{
-		Addr:              o.Address,
+		Addr:              address,
```

Comment:
`Addr` can be dropped as it is not used by  `Serve` and `ServeTLS`, only by `ListenAndServe` and `ListenAndServeTLS` which we do not use.